include/inexor/vulkan-renderer/fps_counter.hpp

```diff
@@ -18,7 +18,7 @@ class FPSCounter {
 
     ~FPSCounter() = default;
 
-    std::optional<std::uint32_t> update();
+    [[nodiscard]] std::optional<std::uint32_t> update();
```

Comment:
Here nodiscard is not usefull. As it updates internal variables of the fps counter, which can be done without using the fps value at the end.

Reply:
ok

Reply:
done.